src/components/FlowSelector.js

```diff
@@ -51,14 +48,14 @@ class FlowSelector extends React.Component {
     return (
       <AppContext.Consumer>
         {({ appCtx, updateContext }) => (
-          <div>
+          <div><center>
```

Comment:
Can we avoid using the `<center>` tag and instead add a className to the div and use that instead?